iphone/cli/hooks/frameworks.js

```diff
@@ -526,11 +533,11 @@ class FrameworkIntegrator {
 	 * @access private
 	 */
 	addFrameworkFileReference(frameworkInfo) {
-		let frameworkPackageName = frameworkInfo.name + '.framework';
-		let fileRefUuid = this._builder.generateXcodeUuid();
+		const frameworkPackageName = frameworkInfo.packageName;
+		const fileRefUuid = this._builder.generateXcodeUuid();
 		this._xobjs.PBXFileReference[fileRefUuid] = {
 			isa: 'PBXFileReference',
-			lastKnownFileType: 'wrapper.framework',
+			lastKnownFileType: frameworkInfo.wrapperName, // FIXME: What if framework is an xcframework?
```

Comment:
In case of xcframework it should be "wrapper.xcframework" .

Reply:
I think the fixme comment can be removed, the correct file type will be returned [here](https://github.com/appcelerator/titanium_mobile/pull/12092/files#diff-db24295025b1208c3a957f26d2358510R903)